index.html

```diff
@@ -267,12 +267,20 @@
   <body>
     <section id='abstract'>
       <p>
-[=Credentials=] are integral to our daily lives; driver's licenses confirm
-our capability to operate motor vehicles, university degrees assert our level
-of education, and government-issued passports permit travel between countries.
-This specification provides a mechanism to express these sorts of
-[=credentials=] on the Web in cryptographically secure, privacy-respecting,
-and machine-verifiable way.
+This specification provides normative and non-normative guidance on
+implementing and managing [=verifiable credentials=] and associated
+cryptographic practices. It emphasizes the importance of understanding and
+updating cryptographic systems and managing private signing keys with limited
+cryptoperiods. It discusses mechanisms for ensuring content integrity of
+linked external resources and highlights the risks of unsigned claims.
+Strategies are provided to mitigate man-in-the-middle (MITM), replay, and
+spoofing attacks, and to address issues related to credential atomization,
+validity periods, and device security. This specification also covers
+acceptable use of credentials, warns against code injection risks, and
+underscores the need for accessibility and internationalization
+considerations, advocating for a data-first approach and adherence to
+internationalization standards to ensure correct rendering of
+multilingual text.
```

Comment:
-1 to changing the abstract, this was tuned over many years, I don't think the rewrite is better.

Reply:
The existing Abstract *isn't* an abstract. It's an introductory paragraph, which I think is confirmed by its complete reproduction in the Introduction section.

If WG preference is to retain the existing paragraph as the Abstract, WG must also decide what to do about the final sentence of the paragraph in its Introduction incarnation, from which I think this entire paragraph should then be removed.

Reply:
Moving Abstract revision to #1560
```suggestion
[=Credentials=] are integral to our daily lives: driver's licenses confirm
our capability to operate motor vehicles; university degrees assert our level
of education; and government-issued passports attest to our citizenship when 
traveling between countries. This specification provides a mechanism for
expressing these sorts of [=credentials=] on the Web in a way that is 
cryptographically secure, privacy respecting, and machine verifiable. These
[=credentials=] provide benefits to us when used in the physical world, but
their use on the Web continues to be elusive.
```